src/Plugins/Github.php

```diff
@@ -26,12 +30,17 @@ public function __construct(ChatClient $chatClient, HttpClient $httpClient, KeyV
 
     public function enableForRoom(Room $room, bool $persist = true)
     {
-        repeat(
-            function() use ($room) { return $this->checkStatusChange($room); }, 150000
-        );
+        if (!$this->rooms) {
+            repeat(function () {
+                yield $this->checkStatusChange();
+            }, 150000);
+        }
+
+        $this->rooms[] = $room;
```

Comment:
Here's where there's a change that's actually *required* (the others are just my preference):

`$this->enabledRooms[$room->getIdentifier()->getIdentString()] = $room;`

We will need to handle `disableForRoom()` as well, in which we will remove the target room from the map, and stop the repeater when it's not enabled in any rooms any more.